corehq/apps/app_manager/models.py

```diff
@@ -5311,6 +5369,13 @@ def fetch_file(self, location):
 
         return location, content
 
+    def get_build_langs(self):
+        if self.build_profiles:
+            # return first profile, generated as part of lazy migration
+            return self.build_profiles[self.build_profiles.keys()[0]].langs
```

Comment:
i am fine raising an exception here though also added code to the view function to prevent saving a remote app with more than 1 build profile